From ce45cf8651fabe48a75d8773dbdbf68f3dc77137 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Tue, 15 Jan 2019 19:05:41 +0100
Subject: [PATCH] artificially serialize building of .policy files

This avoids concurrent write access to translation cache by
intltool-merge.

Resolves: #1272485
---
 Makefile.am | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index 40ebbe98ee..995c421b8b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -653,6 +653,16 @@ dist_doc_DATA = \
 
 @INTLTOOL_POLICY_RULE@
 
+# Force serial build of policy files
+# Note that this ignores the fact that almost all of these files are only built optionally,
+# but a change in configure options is extremely unlikely at this point.
+src/core/org.freedesktop.systemd1.policy : src/hostname/org.freedesktop.hostname1.policy
+src/hostname/org.freedesktop.hostname1.policy : src/import/org.freedesktop.import1.policy
+src/import/org.freedesktop.import1.policy : src/locale/org.freedesktop.locale1.policy
+src/locale/org.freedesktop.locale1.policy : src/login/org.freedesktop.login1.policy
+src/login/org.freedesktop.login1.policy : src/machine/org.freedesktop.machine1.policy
+src/machine/org.freedesktop.machine1.policy : src/timedate/org.freedesktop.timedate1.policy
+
 # ------------------------------------------------------------------------------
 
 MANPAGES =
